include/deal.II/lac/block_vector.h

```diff
@@ -517,6 +518,16 @@ namespace internal
   } /* namespace LinearOperator */
 } /* namespace internal */
 
+
+/** mark dealii::BlockVector< Number > as serial vector
```

Comment:
[Here's](https://github.com/dealii/dealii/blob/master/include/deal.II/base/aligned_vector.h#L69) an example I've found (admittedly for a class function)
```
  /**
   * Empty constructor. Sets the vector size to zero.
   */
```
and [here](https://github.com/dealii/dealii/blob/master/include/deal.II/base/utilities.h#L422) for a normal function
```
    /**
     * Return the name of the host this process runs on.
     */
```
So I think that for consistency we should just follow suite.